libevmasm/Assembly.cpp

```diff
@@ -1393,6 +1398,24 @@ LinkerObject const& Assembly::assembleEOF() const
 	m_tagPositionsInBytecode = std::vector<size_t>(m_usedTags, std::numeric_limits<size_t>::max());
 	std::map<size_t, uint16_t> dataSectionRef;
 
+	bool storesAuxData = false;
+	bool loadsAuxData = false;
```

Comment:
These should be inside the loop. Otherwise `true` from the previous section will carry 
over to the next one, and I think that was not the intention here?

Reply:
I think it's ok. We want to make sure that there is no container which stores and loads auxiliary data. So storing container is init container and loading is runtime container.